### ISSUE TITLE
Increasing learning rate

### ISSUE THREAD

**Issue by Hong7Cong** [rmaphoh/RETFound_MAE#8]
While fine-tune pretrained RETFound_cfp on our dataset, I noticed that the learning rate was increasing every batch. Why? Is there any practice behind this? Thanks

![increaselr](https://github.com/rmaphoh/RETFound_MAE/assets/49111748/b87f297d-5c20-40dc-b204-1b89684b198f)

My bash command was:
python -m torch.distributed.launch --nproc_per_node=1 --master_port=48798 main_finetune.py --batch_size 4 --world_size 1 --model vit_large_patch16 --epochs 100 --lr 5e-3 --blr 5e-3 --layer_decay 0.65 --weight_decay 0.05 --drop_path 0.1 --nb_classes 2 --data_path ../../chla_fundus/ --task ./finetune_chla/ --finetune ./RETFound_cfp_weights.pth

**Comment by rmaphoh** [comment]
Hi Hong, thanks for your interest in RETFound. The learning rate is scheduled and updated in the [line](https://github.com/rmaphoh/RETFound_MAE/blob/cf9c34c801f196d5e1b154c3c1697ebe4100a147/engine_finetune.py#L86). This is a practical learning rate scheduler, using warming up (increasing) followed by cosine annealing (decreasing). The curve looks like below.

<img width="268" alt="image" src="https://github.com/rmaphoh/RETFound_MAE/assets/68285473/0c1f56cf-c017-42ff-beca-94a6a8b736c9">

**Comment by rmaphoh** [comment]
This issue has been closed due to no further response.
